custom/up_nrhm/reports/asha_facilitators_report.py

```diff
@@ -9,8 +9,8 @@
 
 def total_rows(report):
     return {
-        "total_under_facilitator": report.total_under_facilitator,
-        "total_with_checklist": report.total_with_checklist
+        "total_under_facilitator": getattr(report, 'total_under_facilitator', 0),
+        "total_with_checklist": getattr(report, 'total_with_checklist', 0)
```

Comment:
@snopoke I had a attribute error when I opened a report and filters were not selected
I'll fix things that you mentioned in other comments in pr with next report.